mod_form.php

```diff
@@ -56,6 +56,28 @@ public function definition() {
             $nexturl = $PAGE->url;
             zoom_fatal_error($errstring, 'mod_zoom', $nexturl, $config->zoomurl);
         }
+        
+        // Get list of schedule for users if supported.
+        // List of users who can use Zoom mod in this class.
+        $moodleusers = get_enrolled_users(context $context, 'mod/zoom:addinstance', 0, $'u.*', 'lastnane');
+        
+        /**
+         * @var $scheduleusers array Array of emails and proper names of Moodle users in this course that can add Zoom meetings, and the user can schedule.
+        */
+        $scheduleusers = [];
+        // This will either be false (they can't) or the list of users they can schedule.
+        if(($canschedule = $service->get_schedule_for_users($USER->email)) !== false) {
+            foreach($canschedule as $zoomuser) {
```

Comment:
Just a little breathing room for that keyword
```suggestion
            foreach ($canschedule as $zoomuser) {
```